app/services/permissions_checker.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class PermissionsChecker
+  def initialize(current_user:, permission_name:, user_id:, friendly_id:, record_id:)
```

Comment:
Maybe `permission_name:, user_id:, friendly_id:, record_id:` should each have default values set

Reply:
Now that I think of it, it isn't needed so nvm